doc/input-output-reference/src/overview/group-thermal-zone-description-geometry.tex

```diff
@@ -4115,7 +4115,9 @@ \subsection{WindowShadingControl}\label{windowpropertyshadingcontrol}
 Note that if both ``Name of Construction with Shading'' and ``Material Name of Shading Device'' are specified, the former takes precedence.
 \end{itemize}
 
-Most Shading Control Types allow you to specify a schedule that determines when the control is active. One example is a control that is active seasonally. For example, to deploy shading only in the summer when the incident solar is high enough, use Shading Control Type = OnIfHighSolarOnWindow with a schedule that is 1 during the summer months and 0 otherwise and specify Shading Control Is Scheduled = YES.
+Most Shading Control Types allow you to specify a schedule that determines when the control is active. One example is a control that is active seasonally. For example, to deploy shading only in the summer when the incident solar is high enough, use Shading Control Type = OnIfHighSolarOnWindow with a schedule that is 1 during the summer months and 0 otherwise and specify Shading Control Is Scheduled = YES. 
+
+To deploy one shading device during one time period and another during another time period for the same window, specify two different WindowShadingControl objects that both reference the same fenestration.  Each of the WindowShadingControl input objects would include on the same window on their lists of fenestration objects, but would reference different schedule input objects. The referenced schedule input objects should have non-zero values specified at different times for each WindowShadingControl. This approach can be used to add shading that is different for different seasons. When more than one WindowShadingControl is used for the same window based on schedule, the type of shading, control type, setpoints, and several other parameters must be the same in the different WindowShadingControl objects, but the shading device and construction can be different.
```

Comment:
I guess just because it wasn't part of the NFP. I think it would require some additional code changes in SolarShading.

Reply:
This seems overly restrictive. In `WindowShadingManager`, once the `activeWindowShadingControl` is set, all of the logic follow the selected WindowShadingControl. So, I don't see any reason why that couldn't have different rules. I don't think it would even need to be the same shading type.